examples/bilingual_document.ipynb

```diff
@@ -0,0 +1,305 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "Using the cached checkpoint of text_sonar_basic_encoder. Set `force` to `True` to download again.\n",
+      "Using the cached checkpoint of text_sonar_basic_decoder. Set `force` to `True` to download again.\n",
+      "Using the cached tokenizer of text_sonar_basic_encoder. Set `force` to `True` to download again.\n"
+     ]
+    }
+   ],
+   "source": [
+    "text_encoder_model = load_sonar_text_encoder_model(\"text_sonar_basic_encoder\", device=device).eval()\n",
+    "text_decoder_model = load_sonar_text_decoder_model(\"text_sonar_basic_decoder\", device=device).eval()\n",
+    "text_tokenizer = load_sonar_tokenizer(\"text_sonar_basic_encoder\")\n",
+    "\n",
+    "text_embedding_pipeline = TextToEmbeddingModelPipeline(text_encoder_model, text_tokenizer)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Similarity of sentences between texts\n",
+    "\n",
+    "We may compute the matrix cosine similarity of each of the sentences in the novel extracts, with each of the sentences present in the translations."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from sklearn.metrics.pairwise import cosine_similarity\n",
+    "\n",
+    "eng_embeddings = text_embedding_pipeline.predict(eng_sentences, source_lang=\"eng_Latn\")\n",
+    "spa_embeddings = text_embedding_pipeline.predict(spa_sentences, source_lang=\"spa_Latn\")\n",
+    "deu_embeddings = text_embedding_pipeline.predict(deu_sentences, source_lang=\"deu_Latn\")\n",
+    "\n",
+    "eng_spa_cos_sim = cosine_similarity(eng_embeddings, spa_embeddings)\n",
+    "eng_deu_cos_sim = cosine_similarity(eng_embeddings, deu_embeddings)\n",
+    "spa_deu_cos_sim = cosine_similarity(spa_embeddings, deu_embeddings)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
```

Comment:
the values themselves are hard to read in that way, you can print shapes of sim matrices and show heatmaps with seaborn